src/Core/MySQL/PacketsPreparedStatements.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <Core/MySQL/IMySQLWritePacket.h>
+
+namespace DB
+{
+namespace MySQLProtocol
+{
+    namespace PreparedStatements
+    {
+        // https://dev.mysql.com/doc/dev/mysql-server/latest/page_protocol_com_stmt_prepare.html#sect_protocol_com_stmt_prepare_response_ok
```

Comment:
The referenced page contains:

```
if (packet_lenght > 12) {
--
int<2> | warning_count | Number of warnings
if capabilities & CLIENT_OPTIONAL_RESULTSET_METADATA {
int<1> | metadata_follows | Flag specifying if metadata are skipped or not. See enum_resultset_metadata
} – CLIENT_OPTIONAL_RESULTSET_METADATA {
} – packet_lenght > 12
```

In other words, field "warnings_count" is described as conditional (only if package_length > 12). What is "package_length" in this context and is it always > 12? Asking because we currently always send "warning_count".